mediapipe/tasks/ios/text/text_classifier/sources/MPPTextClassifier.mm

```diff
@@ -0,0 +1,98 @@
+#import "mediapipe/tasks/ios/common/utils/sources/MPPCommonUtils.h"
+#import "mediapipe/tasks/ios/common/utils/sources/NSString+Helpers.h"
+#import "mediapipe/tasks/ios/core/sources/MPPTaskInfo.h"
+#import "mediapipe/tasks/ios/core/sources/MPPTextPacketCreator.h"
+#import "mediapipe/tasks/ios/text/core/sources/MPPTextTaskRunner.h"
+#import "mediapipe/tasks/ios/text/text_classifier/utils/sources/MPPTextClassifierOptions+Helpers.h"
+#import "mediapipe/tasks/ios/text/text_classifier/utils/sources/MPPTextClassifierResult+Helpers.h"
+
+#include "mediapipe/tasks/cc/components/containers/proto/classifications.pb.h"
+
+#include "absl/status/statusor.h"
+
+namespace {
+using ::mediapipe::Packet;
+using ::mediapipe::tasks::components::containers::proto::ClassificationResult;
+using ::mediapipe::tasks::core::PacketMap;
+}  // namespace
+
+static NSString *const kClassificationsStreamName = @"classifications_out";
+static NSString *const kClassificationsTag = @"CLASSIFICATIONS";
+static NSString *const kTextInStreamName = @"text_in";
+static NSString *const kTextTag = @"TEXT";
+static NSString *const kTaskGraphName = @"mediapipe.tasks.text.text_classifier.TextClassifierGraph";
+
+@interface MPPTextClassifier () {
+  /** iOS Text Task Runner */
+  MPPTextTaskRunner *_textTaskRunner;
+}
+@end
+
+@implementation MPPTextClassifier
+
+- (instancetype)initWithOptions:(MPPTextClassifierOptions *)options error:(NSError **)error {
+  MPPTaskInfo *taskInfo = [[MPPTaskInfo alloc]
+      initWithTaskGraphName:kTaskGraphName
+               inputStreams:@[ [NSString stringWithFormat:@"%@:%@", kTextTag, kTextInStreamName] ]
+              outputStreams:@[ [NSString stringWithFormat:@"%@:%@", kClassificationsTag,
+                                                          kClassificationsStreamName] ]
+                taskOptions:options
+         enableFlowLimiting:NO
+                      error:error];
+
+  if (!taskInfo) {
+    return nil;
+  }
+
+  _textTaskRunner =
+      [[MPPTextTaskRunner alloc] initWithCalculatorGraphConfig:[taskInfo generateGraphConfig]
+                                                         error:error];
+  
+  if (!_textTaskRunner) {
+    return nil;
+  }                                                      
+  
+  self = [super init];
+
+  return self;
+}
+
+- (instancetype)initWithModelPath:(NSString *)modelPath error:(NSError **)error {
+  MPPTextClassifierOptions *options = [[MPPTextClassifierOptions alloc] init];
+
+  options.baseOptions.modelAssetPath = modelPath;
+
+  return [self initWithOptions:options error:error];
+}
+
+- (nullable MPPTextClassifierResult *)classifyWithText:(NSString *)text error:(NSError **)error {
+  Packet packet = [MPPTextPacketCreator createWithText:text];
+
+  std::map<std::string, Packet> packet_map = {{kTextInStreamName.cppString, packet}};
+  absl::StatusOr<PacketMap> status_or_output_packet_map = [_textTaskRunner process:packet_map];
```

Comment:
In an earlier review I got feedback that said that we should use camelCase in Objective-C code even for C++ types.

Reply:
Updated to camel case in all .mm files.